test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -224,12 +224,22 @@ def api_client
 
   # GRPC logging mock that successfully logs the records.
   class GRPCLoggingMockService < Google::Logging::V1::LoggingService::Service
-    def initialize(requests_received)
+    def initialize(stub_params, requests_received)
       super()
       @requests_received = requests_received
+      @expected_requests = stub_params.clone
     end
 
     def write_log_entries(request, _call)
+      matched = @expected_requests.take_while do |expected|
+        "projects/#{expected[:project_id]}/logs/#{expected[:log_name]}" \
+          == request.log_name
```

Comment:
The non-grpc side checks both the `project_id` and log_name` though. Might as well be equivalent?